### ISSUE TITLE
Add an option to disable `repos.createCommitComment`

### ISSUE THREAD

**Issue by odanado** [nwtgck/actions-netlify#126]
Currently, two comments are generated by `repos.createCommitComment` and `issues.createComment`.
e.g.
- `repos.createCommitComment`
  - https://github.com/nwtgck/ssh-keygen-web/commit/1337c7e04e4bad54e433303a1c1ffad7cd22ee50#comments
- `issues.createComment`
  - https://github.com/nwtgck/ssh-keygen-web/pull/82#commitcomment-38963922

The same comments confuse developers and overwhelm their email boxes. 
I think it would be useful to be able to control the type of comments in the options.

**Comment by nwtgck** [comment]
@odanado Thank you very much for your suggestion.

> overwhelm their email boxes

Yes! I think so too. I'll make the options for opt-out comment generation by the bot.

## Workaround

If you use sieve for filtering, the following script may help you.

```sieve
/* Move to "GitHub Actions Bot" */
/* NOTE: Currently comment notification email from nwtgck/actions-netlify is noisy. I'd like to check the body of mails, but require ["body"]; could not be done in ProtonMail. */
if header :is "X-Github-Sender" "github-actions[bot]" {
  fileinto "GitHub Actions Bot";
}
```

**Comment by nwtgck** [comment]
I made a solution for this issue in #129.

**Comment by nwtgck** [comment]
@odanado 

I released 1.0.13 for this feature. `enable-commit-comment: false` should be what you want.
https://github.com/nwtgck/actions-netlify/blob/38c6c209b5bb9ffa4f5c60e20882e4d7206599e2/CHANGELOG.md#1013---2020-05-09

**Comment by odanado** [comment]
Thank you! :+1:
